tests/integration/test_metric_meta.py

```diff
@@ -0,0 +1,99 @@
+import json
+import time
+
+from sentry_sdk.envelope import Envelope, Item, PayloadRef
+from datetime import datetime, timezone, time as dttime
+
+
+def assert_location(expected, actual):
+    # Compare without the 'type' which is not persisted into storage.
+    assert {k: expected[k] for k in expected.keys() - {"type"}} == actual
```

Comment:
I think this should be equivalent"
```suggestion
    # Compare without the 'type' which is not persisted into storage.
    expected.pop('type')
    assert expected == actual
```

Reply:
This modifies the passed in reference `expected` so we'd at least have to make a copy of it